client/src/components/USBController.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+
+import useInterval from 'use-interval'
+
+import Button from './Button'
+import Text from './Text'
+import { isPresent, isMounted, doMount, doUnmount, isAvailable } from '../lib/usbstick'
+
+const USBController = () => {
+  const available = isAvailable()
+  const [present, setPresent] = useState(false)
+  const [mounted, setMounted] = useState(false)
+
+  useInterval(() => {
+    (async () => {
+      const p = await isPresent()
+      setPresent(p)
+      if (p)
+	setMounted(await isMounted())
+    })()
+  }, available ? 1000 : null);  
+  
+  if (!available) {
+    return (<React.Fragment></React.Fragment>)
+  }
+
+  if (!present) {
+    return <Text>No USB</Text>
+  }
+  
+  return (
+    <React.Fragment>
+      {mounted ? (
+	<Button small onPress={doUnmount}>Eject USB</Button>
+      ) : (
+	<Button small onPress={doMount}>Mount USB</Button>
```

Comment:
nit: Your linting doesn't look right. I don't think the hooks are all set up properly in this repo yet.

Reply:
OK, gonna push this one through but will look separately at adding linting.